### PR TITLE
feat: implement on RainbowFirebaseApp `initialize` prop

### PR DESCRIPTION
**Pull request by HellWolf93**
fix: #38

### CODE REVIEW

File: packages/app/src/components/App/index.js [hunk 1]
```diff
@@ -57,6 +73,8 @@ RainbowFirebaseApp.propTypes = {
     translations: PropTypes.object,
     /** Object with your application Redux reducers. */
     reducers: PropTypes.object,
+    /** A funtion to initialize the app. AppSpiner will be visible while this function is running. */
```
Comment (maxxgreene) [inline packages/app/src/components/App/index.js:76]: you have a typo `funtion` --> `function`. Also, we should specify it must be an async function

File: packages/app/src/components/App/index.js [hunk 2]
```diff
@@ -66,6 +84,7 @@ RainbowFirebaseApp.defaultProps = {
     translations: {},
     children: null,
     reducers: {},
+    initialize: () => {},
```
Comment (maxxgreene) [inline packages/app/src/components/App/index.js:87]: if there is a default value it means there a function that will execute by default, is that right?

File: packages/app/src/components/App/index.js [hunk 3]
```diff
@@ -24,12 +26,26 @@ const RainbowFirebaseApp = (props) => {
...
+        async function initializeApp() {
+            if (initialize) {
+                setLoading(true);
+                await initialize();
+                setLoading(false);
+            }
+            setIsInitializing(false);
+        }
+        initializeApp();
+    }, [initialize]);
```
Comment (maxxgreene) [inline packages/app/src/components/App/index.js:39]: it shouldn't depend on `initialize` prop, the `useEffect` callback function should be invoked once, the first time the component renders

**Comment by TahimiLeonBravo** [comment]
@HellWolf93 please review the conflicts here

**Comment by HellWolf93** [comment]
> @HellWolf93 please review the conflicts here

Fixed

**Pull request merged**